src/svxlink/contrib/module_txfan/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# The name of the module without the Module prefix
+set(MODNAME TxFan)
+
+# Install targets
+install(FILES Module${MODNAME}.tcl DESTINATION ${SVX_SHARE_INSTALL_DIR}/modules.d)
+install(FILES ${MODNAME}.tcl DESTINATION ${SVX_SHARE_INSTALL_DIR}/events.d)
+install_if_not_exists(Module${MODNAME}.conf
+  ${SVX_SYSCONF_INSTALL_DIR}/svxlink.d
+  )
```

Comment:
Missing newline at end of file

Reply:
my source has the newline, not sure what to do here

Reply:
Yes, looks good now. Don't know if I misread the diff or if you fixed it earlier